internal/session/repository_session.go

```diff
@@ -249,6 +255,8 @@ func (r *Repository) ListSessions(ctx context.Context, opt ...Option) ([]*Sessio
 	case db.AscendingOrderBy:
 		withOrder = "order by create_time asc"
 	case db.DescendingOrderBy:
+		fallthrough
```

Comment:
With the new view; the default ordering was being affected and breaking unit tests that expected this ordering. I set it to default, but am unsure if this is the preferred fix.

Reply:
It's a reasonable default.  Ty for pointing this out.